requirements.txt

```diff
@@ -1,11 +1,11 @@
 # setup.py install_requires
 # any changes here should also be reflected in setup.py "install_requires"
-boto3 == 1.24.59
-numpy == 1.23.4
-pandas == 1.5.2
+boto3 == 1.20.24
```

Comment:
Glue uses Python 3.9, did we test this on Glue?

Reply:
I haven't encountered any issues with these dependencies on Glue.